packages/netstring/writer.js

```diff
@@ -6,20 +6,8 @@ const COMMA = ','.charCodeAt(0);
 const encoder = new TextEncoder();
 
 /**
- * @template T
- * @template U
- * @template V
- * @typedef {{
```

Comment:
Do you know why this JSDoc typedef for `Stream` didn't work?

Reply:
It worked fine but `import` was less redundant.